docs: add latest product updates and refactor docs (#548)
fleek-platform/website: src/content/guides/fleek-ai-agents-api/index.md

To use the API, you need an X-Api-Key. Follow these steps to retrieve it:

### 1. Generate Your X-Api-Key

To get an API key:

1. Go to the [fleek.xyz/eliza](https://fleek.xyz/eliza/) page. Click on the "Account settings" button at the bottom of the page.

> 2. The "API tokens" tab shows up with all the API keys you have created.

Reviewer: ```suggestion
2. The "API tokens" tab lists all your created API keys.
```

![Eliza page tokens](./eliza-page-tokens.png)

> 3. To get an API key from the Eliza deployment page, click on the "Create API token" button and a modal appears.

Reviewer: ```suggestion
3. To get an API key from the Eliza deployment page, click on the "Create API token" button.
```

> 4. Type in the name of the API key you want to create and click on the "Create API token" button.

Reviewer: ```suggestion
4. Type the name of the API key you want to create and press the "Create API token" button.
```
